frappe/integrations/doctype/slack_webhook_url/slack_webhook_url.py

```diff
@@ -40,7 +42,7 @@ def send_slack_message(webhook_url, message, reference_doctype, reference_name):
 			]
 		}
 	]
-	data = {"text": message, "attachments": attachments}
+	data = {"text": message, "attachments": attachments if show_link != 0 else []}
```

Comment:
Added `!= 0` explicitly to avoid disabling in case of `None` for existing records.

Reply:
Isn't it better to have a migration that set `show_document_link ` to 1 for all existing records?

Reply:
setting default field to 1 takes care of migration... will change this check to usual `if show_link`